src/main/java/com/splunk/logging/HttpEventCollectorLogbackAppender.java

```diff
@@ -79,6 +80,10 @@ public void start() {
         if (_messageFormat != null)
             metadata.put(MetadataTags.MESSAGEFORMAT, _messageFormat);
 
+        if ("raw".equalsIgnoreCase(_type)) {
+
```

Comment:
I think you're missing a calculation of _batchingConfigured here

Reply:
Thanks!